Sources/Jobs/Schedule.swift

```diff
@@ -17,6 +17,10 @@ public class Schedule<K: KeyChain, T: Job>: Configuration {
     ///     - on: Crontab as a String.
     ///     - keyPath: Associates a `Job` for later retrieval.
     public init(_ job: T, on cronTrigger: String, _ keyPath: KeyPath<K, T>) {
+        // Activate any `ObservedObject`s on the job.
+        var job = job
+        activate(&job)
```

Comment:
I think it is better to remove these lines an keep only one activation step in the `Scheduler`. 

I'm not really sure why I already add the job to the `Environment` in the initializer. I will fix this in #180

Reply:
Ok...seems not to break anything.

Reply:
I will handle this on my branch 😄